### DISCUSSION TITLE
Codesandbox of an AR experience using hit testing (useHitTest) and other hooks (Interactive, useXR) provided by react-xr

### DISCUSSION THREAD

**Discussion by mohitkumartoshniwal** [pmndrs/xr#265] [Show and tell] [answered]
Hi, First of all, thanks for this useful library. I had quite a lot of fun using this library to build some experiences like  

- https://xr-character-configurator.vercel.app/ 
- https://xr-tic-tac-toe.vercel.app/

But I had to do a lot of work to get them right in the first place since there were very less resources online, and whatever stackoverflow answers I could find, they were quite outdated. It took me quite some time to make them work after a lot of trial and errors.

And since I felt this library was quite underrated with regards to its AR features, I made a [Youtube playlist](https://youtube.com/playlist?list=PLpM_sf_d5YTPXeVp4cmgN_cNBj9pNTEmZ) for the same. 

Other than that I have also created a [sandbox](https://codesandbox.io/s/frosty-shape-yh1k9w) example which shows the use of useHitTest and other hooks provided by this package to build an AR experience which I will be covering in the youtube playlist. 

Hope to have this sandbox as part of the examples section in the README.md so that it helps anyone who stumbles upon this library and would like to build an AR experience with it and I am open to making any modifications to the sandbox required for the same.

Lastly, I was not sure whether I should create an issue for this since it was a README.md change, hence started the discussion.

**Comment by bbohlender** [comment, marked as answer]
Hey @mohitkumartoshniwal
I am currently going through all the open questions, issues, etc.
Once hit test is propertly implemented in v6, I will add a simple example to the docs.
Thanks a lot for your videos on youtube. Maybe we can use the documentation and dicuss about a coherent set of tutorials for react-three/xr that people can follow from start to finish. Would definedly offer my help with that and feel free to use the documentation as a resource for that :)

**Reply by mohitkumartoshniwal** [reply]
Yeah, I have started thinking in that direction where it will be easy for people to follow from start to finish since I feel v6 has made it more easier to work in XR, both AR and VR. And I obviously will be referring the documentation and the examples as well, they are quite good. I would love to have your inputs as well at that point of time 😄.
